website/docs/reference/resource-configs/strategy.md

```diff
@@ -129,6 +214,7 @@ This is a **required configuration**. There is no default value.
 
 {% endsnapshot %}
 ```
+</VersionBlock>
 
 ### Advanced: define and use custom snapshot strategy
```

Comment:
@dbeatty10 is this rigth?

Reply:
I didn't try this out yet, but it looks right visually 👍